azext_edge/edge/params.py

```diff
@@ -439,12 +439,16 @@ def load_iotops_arguments(self, _):
         context.argument(
             "group_id",
             options_list=["--group-id", "--gid"],
-            help="ID of consumer group that the data flow uses to read messages " "from the Kafka topic.",
+            help="ID of consumer group that the data flow uses to read messages "
+            "from the Kafka topic.",
         )
         context.argument(
             "partition_strategy",
             options_list=["--partition-strategy", "--ps"],
-            arg_type=get_enum_type(KafkaPartitionStrategyType, default=KafkaPartitionStrategyType.DEFAULT.value),
+            arg_type=get_enum_type(
```

Comment:
What is the line length of your auto formatter? A bunch of other formatting changes are similar noops.